app.js

```diff
@@ -4,7 +4,35 @@ const { OpenAI } = require('openai');
 
 config();
 
-/** Initialization */
+// LLM system prompt
+const DEFAULT_SYSTEM_CONTENT = `You're an assistant in a Slack workspace.
+Users in the workspace will ask you to help them write something or to think better about a specific topic.
+You'll respond to those questions in a professional way.
+When you include markdown text, convert them to Slack compatible ones.
+When a prompt has Slack's special syntax like <@USER_ID> or <#CHANNEL_ID>, you must keep them as-is in your response.`;
+
+// Feedback buttons block
```

Comment:
```suggestion
/**
 * Feedback buttons to include with messages.
 *
 * @type {import("@slack/bolt").types.ContextActionsBlock}
 */
```

🚧 suggestion: We might want to use explicit typing for global variables like this to make what we expect clear.

🏷️ suggestion(non-blocking): I also changed wording of the comment but am unsure of alternatives for "feedbackBlock" as a variable. I think matching the language used on the backend makes this more clear overall though - perhaps "contextActionsBlock" is alright?

Reply:
Absolutely love this comment block @zimeg! Let's bring it in!